src/VisualStudioExtension/test/Tanzu.Toolkit.VisualStudioExtension.Tests/Tanzu.Toolkit.VisualStudioExtension.Tests.csproj

```diff
@@ -15,6 +15,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Huh, so is Newtonsoft still used in the VS project then?

Reply:
Oh I have a vague memory of Newtonsoft being a dependency of another package (maybe Community VS Toolkit?) that was specific to the package proj iirc... maybe that's what's going on?

Reply:
Pretty sure `MSTest.TestAdapter` has a ref, which conflicts with one from somewhere else. Long story short, this ref is only to resolve the conflict